public/app/features/browse-dashboards/api/browseDashboardsAPI.ts

```diff
@@ -57,6 +57,7 @@ interface ImportOptions {
 
 interface RestoreDashboardArgs {
   dashboardUID: string;
+  targetFolder: string;
```

Comment:
minor nit: is this a UID as well? if so, maybe we should name it like `targetFolderUID`